torch/_decomp/decompositions_for_jvp.py

```diff
@@ -107,7 +107,7 @@ def trace(self: Tensor) -> Tensor:
 def log_sigmoid_forward(self: Tensor) -> Tuple[Tensor, Tensor]:
     min = torch.minimum(self.new_zeros(()), self)
     z = torch.exp(-torch.abs(self))
-    if self.is_cuda:
+    if self.is_cuda or self.is_xpu:
```

Comment:
Add `is_xpu` to tensor doc, refer to https://github.com/pytorch/pytorch/blob/6e61ff4fd31703ef4488ec84910eb420d858a1c4/torch/_tensor_docs.py#L6688

Reply:
There are `is_xpu` in this file. ref:https://github.com/pytorch/pytorch/blob/main/torch/_tensor_docs.py#L6716